schedule/yast/autoyast_home_encrypted.yaml

```diff
@@ -0,0 +1,75 @@
+---
+name: autoyast_home_encrypted
+description: |
+  Gnome installation with separate /home partition encrypted
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader_start
+  - autoyast/installation
+  - installation/decrypt_partition
```

Comment:
can we use here existing module to decrypt? https://openqa.suse.de/tests/4186828#step/boot_encrypt/1

Reply:
i can and make sense. What i am waiting for is an answer from @OleksandrOrlov because the name of the existing one is boot_encrypt and i wonder if i should rename it to reflect the generalization of the purpose.